CloneResponsive/index.html

```diff
@@ -0,0 +1,432 @@
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap"
+    rel="stylesheet">
+  <title>Clone</title>
+</head>
+
+<body>
+  <section class="main__container">
+    <section class="header">
+      <div class="logo">
+        <img src="./images/Logo.svg" alt="logo" class="logo__icon" />
+      </div>
+      </div>
+      <div class="header__main">
+        <div class="header__main__searchbar">
+          <div class="header__main__searchbar__icon">
+            <img src="./images/header-Icons/search.svg" alt="search icon">
+          </div>
+          <div class="header__main__searchbar__placeholder">
+            <input type="search" placeholder="Search..." id="search-input" size=100>
+          </div>
+        </div>
+        <div class="header__main__notification">
+          <div class="header__main__notification__icon">
+            <img src="./images/header-Icons/notification.svg" alt="notification indicator">
+          </div>
+        </div>
+        <div class="header__main__calendar">
+          <div class="header__main__calendar__icon">
+            <img src="./images/header-Icons/calendar.svg" alt="calendar icon">
+          </div>
+        </div>
+        <div class="header__main__tiles">
+          <div class="header__main__tiles__icon">
+            <img src="./images/header-Icons/tiles.svg" alt="app tiles">
+          </div>
+        </div>
+        <div class="header__main__avatar">
+          <div class="header__main__avatar__icon">
+
+            <img src="./images/header-Icons/green-indicator.svg" alt="status indicator"
+              class="header__main__avatar__icon__status">
+          </div>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="content__container">
+      <div class="content__container__upgrade">
+        <div class="content__container__upgrade__description">
+          <h3>Unlock premium stats</h3>
+          <p>Get up to 10TB of storage for a limited time</p>
+        </div>
+        <div class="content__container__upgrade__button">
+          <div class="content__container__upgrade__button__icon">
+            <img src="./images/content-Icons/lightning.svg" alt="lightning">
+          </div>
+          <div>
+            Upgrade
+          </div>
+
+        </div>
+      </div>
+      <div class="content__container__stats">
+        <div class="content__container__stats__revenue">
+          <p>
+            Revenue
+          </p>
+          <h2>
+            $56,945
+          </h2>
+          <div class="content__container__stats__revenue__change">
+            <div class="content__container__stats__revenue__change__current">
+              +45%
+            </div>
+            <div class="content__container__stats__revenue__change__previous">
+              From 4.6%
+            </div>
+          </div>
+
+        </div>
+        <div class="content__container__stats__users">
```

Comment:
Class doesn't describe it content. it describes the visual representation.
For example you see that blocks with Revenue, Users, New Signups and Retention looks the same. It means that the should have the same class